mlir/test/Target/LLVMIR/frame-pointer.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+
+llvm.func @frame_pointer_func() frame_pointer="non-leaf" {
+  // CHECK-LABEL: define void @frame_pointer_func
+  // CHECK: attributes #{{.*}} = { "frame-pointer"="non-leaf" }
```

Comment:
```suggestion
  // CHECK-LABEL: define void @frame_pointer_func() #[[ATTRS:\d+]]
  // CHECK: attributes #[[ATTRS]] = { "frame-pointer"="non-leaf" }
```
you can make the test more robust to future changes here by making sure that the attribute set corresponds to the attribute set of the function

Reply:
Small update on this topic, ATTRS cannot be defined in CHECK-LABEL and we have to use CHECK-SAME just below CHECK-LABEL, as well \d is not working but [0-9]+ works. Will update in next commit, thanks!